Gopkg.toml

```diff
@@ -37,6 +37,10 @@
   branch = "master"
   name = "github.com/marstr/goalias"
 
+[[constraint]]
+  revision = "8bdf7d1a087ccc975cf37dd6507da50698fd19ca"
```

Comment:
Is there a reason the revision is to a commit and not a semver?

Reply:
Yep, so that it's even more locked down than a semver. This way, even if I make the same mistake Satori did, folks using `dep` won't get broken.

Reply:
If you'd prefer, I can move it to a semver

Reply:
Well I do think that semvers are easier to reason about however if we think this is a short-term fix then I think it's fine to leave it as-is.

Reply:
I'll add a note to evaluate this in #949